### ISSUE TITLE
self::has not working in model

### ISSUE THREAD

**Issue by Vinze** [topclaudy/compoships#16]
When querying relationship existence from within a model itself thats using the Compoships trait the `has()` function doesn't work. See the following example:

```php
class Bookmark extends Model {

    use \Awobaz\Compoships\Compoships;

    public static function folders() {
        return self::has('product')
            ->orderBy('folder')
            ->groupBy('folder')
            ->pluck('folder', 'folder');
    }
}
```

This creates the following error: `Non-static method App\Models\Bookmark::has() should not be called statically`

I'm using Laravel 5.5 and Compoships 1.0.8.

**Comment by topclaudy** [comment]
Fixed in 1.0.9.

**Comment by Vinze** [comment]
Great, thanks!
